components/post_view/post_info/post_info.tsx

```diff
@@ -194,14 +194,13 @@ export default class PostInfo extends React.PureComponent<Props, State> {
         const hover = this.props.hover || this.state.showEmojiPicker || this.state.showDotMenu || this.state.showOptionsMenuWithoutHover;
 
         const showCommentIcon = fromAutoResponder ||
-        (!isSystemMessage && (isMobile || hover || (!post.root_id && Boolean(this.props.replyCount)) || this.props.isFirstReply));
+        (!isSystemMessage && (isMobile || hover || (!post.root_id && Boolean(this.props.hasReplies)) || this.props.isFirstReply));
         const commentIconExtraClass = isMobile ? '' : 'pull-right';
         let commentIcon;
         if (showCommentIcon) {
             commentIcon = (
                 <CommentIcon
                     handleCommentClick={this.props.handleCommentClick}
-                    commentCount={collapsedThreadsEnabled ? undefined : this.props.replyCount}
```

Comment:
When CRT is "on" we do not show the comment count on the icon, 
maybe we should move this logic to `components/post_view/comment_icon/index.ts`

Reply:
Good catch. Let me check if that's needed